### PR TITLE
RNs - SHA256 binary renamed

### PR DESCRIPTION
**Pull request by codyhoag**

### CODE REVIEW

**Comment by thiagoalessio** [comment]
/lgtm

cc @sosiouxme and @jasinner

File: release_notes/ocp-4-6-release-notes.adoc [hunk 1]
```diff
@@ -1394,6 +1394,14 @@ The `oc adm must-gather` command no longer collects audit logs by default. You m
 $ oc adm must-gather -- /usr/bin/gather_audit_logs
 ----
 
+[discrete]
+[id="ocp-4-6-binary-sha256sum-txt-renamed-for-ocp-releases"]
+==== Binary `sha256sum.txt` file has been renamed for {product-title} releases
```
Comment (bergerhoffer) [inline release_notes/ocp-4-6-release-notes.adoc:1399]: Should this be `sha256sum.txt.sig`, as the first sentence specifies?

**Review by bergerhoffer — COMMENTED** [review]
Just one question, LGTM otherwise if it's right!

File: release_notes/ocp-4-6-release-notes.adoc (hunk 1 above)
Comment (codyhoag) [reply release_notes/ocp-4-6-release-notes.adoc:1399]: Yeah, good point. :+1:

**Review by bergerhoffer — APPROVED** [review]

**Pull request merged**